upgrades/2025_04_10_162213_remove_storage_symlink.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+use Illuminate\Support\Facades\Artisan;
+use ProcessMaker\Upgrades\UpgradeMigration as Upgrade;
+
+class RemoveStorageSymlink extends Upgrade
+{
+    /**
+     * Run any validations/pre-run checks to ensure the environment, settings,
+     * packages installed, etc. are right correct to run this upgrade.
+     *
+     * Throw a \RuntimeException if the conditions are *NOT* correct for this
+     * upgrade migration to run. If this is not a required upgrade, then it
+     * will be skipped. Otherwise the exception thrown will be caught, noted,
+     * and will prevent the remaining migrations from continuing to run.
+     *
+     * Returning void or null denotes the checks were successful.
+     *
+     * @return void
+     *
+     * @throws RuntimeException
+     */
+    public function preflightChecks()
+    {
+        //
+    }
+
+    /**
+     * Run the upgrade migration.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Artisan::call('storage:unlink');
```

Comment:
This breaks the access to some publc files like profile avatar

![image](https://github.com/user-attachments/assets/a8bbbfd9-6eb3-4045-bf21-9cc78374ad44)